src/Iris/Tool.hs

```diff
@@ -82,24 +83,38 @@ defaultToolSelector = ToolSelector
 data ToolCheckResult
     {- |
 
+    @since 0.0.0.1
```

Comment:
I prefer to use `@since x.x.x.x` for new unreleased changes. I'll update CONTRIBUTING guide, mentioning this workflow. But the motivation is that the new version is not known at the moment of introducing new change. So we can write `x.x.x.x` and later replace all such places with a single shell command.

```suggestion
    @since x.x.x.x
```

In this case, it's a breaking change, so the new Iris version already will be `0.1.0.0`. Still, it's better to follow some practices 😌